folding/tests/test_decomposable_folding.rs

```diff
@@ -385,7 +385,11 @@ fn test_decomposable_folding() {
         let FoldingOutput {
             folded_instance,
             folded_witness,
-            ..
+            t_0: _,
```

Comment:
It is a personal taste to explicitly unwrap it totally. It might be a pain when we add a field, but it gives information on the API. If there are many changes when unwrapping a structure, it means the API is not well implemented.